src/login/LoginFailure.jsx

```diff
@@ -161,60 +172,56 @@ const LoginFailureMessage = (props) => {
       }
       return (
         <ChangePasswordPrompt
-          redirectUrl={props.loginError.redirectUrl}
+          redirectUrl={context.redirectUrl}
           variant="nudge"
         />
       );
     case REQUIRE_PASSWORD_CHANGE:
       return <ChangePasswordPrompt />;
     case TPA_AUTHENTICATION_FAILURE:
-      errorList = (
-        <p>{formatMessage(messages['login.tpa.authentication.failure'], {
-          platform_name: getConfig().SITE_NAME,
-          lineBreak: <br />,
-          errorMessage: context.errorMessage,
-        })}
+      errorMessage = (
+        <p>
+          {formatMessage(messages['login.tpa.authentication.failure'], {
+            platform_name: getConfig().SITE_NAME,
+            lineBreak: <br />,
+            errorMessage: context.errorMessage,
+          })}
         </p>
       );
       break;
     case INTERNAL_SERVER_ERROR:
     default:
-      errorList = <p>{formatMessage(messages['internal.server.error.message'])}</p>;
+      errorMessage = <p>{formatMessage(messages['internal.server.error.message'])}</p>;
       break;
   }
 
   return (
     <Alert id="login-failure-alert" className="mb-5" variant="danger" icon={Error}>
       <Alert.Heading>{formatMessage(messages['login.failure.header.title'])}</Alert.Heading>
-      { errorList }
+      { errorMessage }
     </Alert>
   );
 };
 
 LoginFailureMessage.defaultProps = {
-  loginError: {
-    redirectUrl: null,
-    errorCode: null,
-    errorMessage: null,
-  },
+  context: {},
 };
 
 LoginFailureMessage.propTypes = {
-  loginError: PropTypes.shape({
-    context: PropTypes.shape({
-      supportLink: PropTypes.string,
-      platformName: PropTypes.string,
-      tpaHint: PropTypes.string,
-      provider: PropTypes.string,
-      allowedDomain: PropTypes.string,
-      remainingAttempts: PropTypes.number,
-      failureCount: PropTypes.number,
-      errorMessage: PropTypes.string,
-    }),
+  context: PropTypes.shape({
+    supportLink: PropTypes.string,
+    platformName: PropTypes.string,
+    tpaHint: PropTypes.string,
+    provider: PropTypes.string,
+    allowedDomain: PropTypes.string,
+    remainingAttempts: PropTypes.number,
+    failureCount: PropTypes.number,
```

Comment:
marking this resolved because its related to https://github.com/openedx/frontend-app-authn/pull/899#discussion_r1458518150 and will hopefully be resolved with that